install.ps1

```
@@ -230,23 +230,340 @@ function Get-StaleBinaryNames {
+    $schemaVersion = Assert-JsonProperty -Object $manifest -Name "schema_version" -Label "schema_version"
+    if ([int]$schemaVersion -ne 2) {
+        throw "product-manifest.json schema_version must be 2"
+    }
+    $contract = Assert-StringField -Value (Assert-JsonProperty -Object $manifest -Name "contract" -Label "contract") -Label "contract"
+    if ($contract -ne "mesh-llm-product-v2") {
+        throw "product-manifest.json contract must be mesh-llm-product-v2"
+    }
+    [void](Assert-StringField -Value (Assert-JsonProperty -Object $manifest -Name "mesh_version" -Label "mesh_version") -Label "mesh_version")
+    [void](Assert-StringField -Value (Assert-JsonProperty -Object $manifest -Name "backend" -Label "backend") -Label "backend")
+
+    $host = Assert-JsonProperty -Object $manifest -Name "host" -Label "host"
```

PSScriptAnalyzer (1.25.0)

[error] The Variable 'host' cannot be assigned since it is a readonly automatic variable that is built into PowerShell, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)